test/unit/helpers/rubygems_helper_test.rb

```diff
@@ -117,6 +117,21 @@ class RubygemsHelperTest < ActionView::TestCase
       assert_equal expected_links, links_to_owners(@rubygem)
       assert links_to_owners(@rubygem).html_safe?
     end
+
+    should "create links to gem owners without mfa" do
+      with_mfa = create(:user, mfa_level: "ui_and_api")
+      without_mfa = Array.new(2) { create(:user, mfa_level: "disabled") }
```

Comment:
Consider using FactoryBot's `create_list`:

```diff
-without_mfa = Array.new(2) { create(:user, mfa_level: "disabled") }
+without_mfa = create_list(:user, 2, mfa_level: "disabled")
```

Reply:
fixed in 6a6d3a3